src/NotFound.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  findByRole, findByText,
+} from '@testing-library/dom';
+import { render } from '@testing-library/react';
+import NotFound from './NotFound';
+
+describe('tests for NotFound page', async () => {
+  let container;
+  beforeEach(() => {
+    container = render(<NotFound />).container;
+  });
+
+  it('should display the image', async () => {
+    const image = await findByRole(container, 'img');
```

Comment:
Do you need to use `findBy...` type of queries (which return a promise) or can you simply use `getBy...` type queries?

Reply:
Updated
Kept the findBy where there were async calls (for alerts)